client/network/src/service.rs

```diff
@@ -889,16 +891,30 @@ impl Metrics {
 			connections_closed_total: register(CounterVec::new(
 				Opts::new(
 					"sub_libp2p_connections_closed_total",
-					"Total number of connections closed, by direction, reason and by being the first or not"
+					"Total number of connections closed, by direction and reason"
 				),
-				&["direction", "reason", "was_first"]
+				&["direction", "reason"]
 			)?, registry)?,
 			connections_opened_total: register(CounterVec::new(
 				Opts::new(
 					"sub_libp2p_connections_opened_total",
-					"Total number of connections opened by direction and by being the first or not"
+					"Total number of connections opened by direction"
 				),
-				&["direction", "is_first"]
+				&["direction"]
+			)?, registry)?,
+			distinct_peers_connections_closed_total: register(CounterVec::new(
+				Opts::new(
+					"sub_libp2p_distinct_peers_connections_closed_total",
+					"Total number of connections closed with distinct peers, by direction and reason"
+				),
+				&["direction", "reason"]
+			)?, registry)?,
+			distinct_peers_connections_opened_total: register(CounterVec::new(
```

Comment:
Same idea: remove the `direction`.

```suggestion
			distinct_peers_connections_opened_total: register(Counter::new(
```